alphadia/workflow/peptidecentric.py

```diff
@@ -15,9 +15,8 @@
 
 # alphadia imports
 from alphadia import fragcomp, plexscoring, utils
-from alphadia.exceptions import NoRecalibrationTargetError
 from alphadia.peakgroup import search
-from alphadia.workflow import base, manager
+from alphadia.workflow import base, manager, searchoptimization
```

Comment:
let's call it just `optimization` or similar, I think the `search` doesn't ad much.

Reply:
OK.